packages/next/test/integration/index.test.js

```diff
@@ -169,14 +175,13 @@ it('should build using server build', async () => {
     output['_next/data/testing-build-id/fallback/[slug].json'].allowQuery
   ).toEqual(['slug']);
   expect(
-    output['_next/data/testing-build-id/fallback/[slug].json'].lambda
-      .operationType
-  ).toBe('ISR');
+    getOperationType(output, '_next/data/testing-build-id/fallback/[slug].json')
+  ).toBe(undefined);
 
   expect(output['fallback/first'].type).toBe('Prerender');
   expect(output['fallback/first'].allowQuery).toEqual([]);
-  expect(output['fallback/first'].lambda.operationType).toBe('ISR');
   expect(output['fallback/first'].sourcePath).toBe('/fallback/[slug]');
+  expect(getOperationType(output, 'fallback/first')).toBe(undefined);
```

Comment:
I think this is supposed to be `SSG` as well. Because of the fallback to the deprecated `Lambda#operationType` and how that defaults to `SSG` for Prerenders, this will work as expected, but it's not great to rely on the deprecated property.

I'm not sure where this path gets added to the outputs in a way that misses the virtual lambda assignment. If someone can help me find that, I'd be happy to update it there.